hack/chains/README.md

```diff
@@ -0,0 +1,99 @@
+# hack/chains
+
+Some bash scripts to help configure and demonstrate [Tekton
+Chains](https://github.com/tektoncd/chains) in an infra-deployments managed
+cluster.
+
+## To set up an environment from scratch
+
+### Bootstrap
+
+For CRC users:
+
+    crc delete
+    crc start
+    `crc console --credentials | tail -1 | cut -d\' -f2`
+
+Then (assuming you're in this branch on your own fork):
+
+    hack/bootstrap-cluster.sh preview
+
+A "go make some coffee" one liner:
+
+    cd $(git rev-parse --show-toplevel); crc delete; crc start; `crc console --credentials | tail -1 | cut -d\' -f2`; hack/bootstrap-cluster.sh preview
+
+Wait a while until you see mostly healthy/synced at [Argo CD](https://openshift-gitops-server-openshift-gitops.apps-crc.testing/applications).
+
+(I have degraded 'has' and 'spi' related to GitHub auth and I don't think it
+matters for the basic pipeline functionality needed to do builds and
+demonstrate chains.)
+
+### Create a key-pair signing secret for chains
+
+    cd hack/chains
+    ./create-signing-secret.sh
+
+### Apply some CA cert related hacks to make SSL work for the internal registry
```

Comment:
Nitpick for later: This title could be revised since it alludes to some other CA cert hacks that aren't there any more.